examples/seismic/skew_self_adjoint/test_wavesolver_iso.py

```diff
@@ -98,18 +79,10 @@ def test_adjoint_F(self, shape, dtype, so):
         Test the forward modeling operator by verifying for random s, r:
             r . F(s) = F^T(r) . s
         """
-        omega = 2.0 * np.pi * fpeak
-        b, v, time_axis, src_coords, rec_coords = \
-            default_setup_iso(npad, shape, dtype, tmax=tmax)
-        solver = SsaIsoAcousticWaveSolver(npad, qmin, qmax, omega, b, v,
-                                          src_coords, rec_coords, time_axis,
-                                          space_order=so)
-        src1 = RickerSource(name='src1', grid=v.grid, f0=fpeak, npoint=1,
-                            time_range=time_axis)
-        src1.coordinates.data[:] = src_coords[:]
-
-        rec1 = Receiver(name='rec1', grid=v.grid, time_range=time_axis,
-                        coordinates=rec_coords)
+        solver = acousticssa_setup(shape=shape, dtype=dtype, space_order=so)
```

Comment:
I still think that acousticssa is barely readable, but fine

Reply:
added extra underscore